app/components/project-recommendation-new.js

```diff
@@ -0,0 +1,126 @@
+import Component from '@ember/component';
+import { inject as service } from '@ember/service';
+import { action } from '@ember/object';
+
+// Creates a new Borough Peesident Recommendation associated with one or more Actions.
+// Borough President Recommendations do not contain voting attributes.
+function createBoroughPresidentRecForActions(thisRec) {
+  const newRec = thisRec.store.createRecord('borough-president-recommendation', {
+    // general
+    user: thisRec.user,
+    // assume all actions for now
+    // TODO: filter actions by user's selection
+    actions: thisRec.project.get('actions'),
+    recommendation: 'Approved with Modifications/Conditions',
+    consideration: 'My awesome memo.',
+    voteLocation: '123 Boston Ave, Brooklyn, NY 11111',
+    dateReceived: new Date('2015-12-12T03:24:00'),
+    dateVoted: new Date('2014-12-17T01:23:45'),
+    isActive: 'active',
+    status: 'Submitted',
+    docketDescription: 'My awesome docket description.',
+  });
+
+  newRec.save();
+}
```

Comment:
I'm a little confused on the test data here... shouldn't this be handled through Mirage? And the dateReceived might even be something the server generates, since it's getting saved in this function immediately.

Reply:
Maybe that's what this refers to? ->

> Delete the stub templates

Reply:
I wanted  the `547-` branch to focus on the logic around creating and saving the recommendation models. I'm planning on hooking up the `project-recommendation-new` component to Mirage Project, User and Action data in the next PR issue 533, the recommendation form templates.

Reply:
> > Hi! Great work... a couple of things:
> > 
> > * This should probably be 2 or 3 separate pull requests. I don't understand why the models and mirage configuration have to go alongside the recommendations component
> > * The recommendations component has a bunch of static data and odd lines (`user = this.store.findRecord...`), but mirage is setup and available, no?
> > * The tests are mostly boilerplate 'it renders' but looks like there are some real ones?
> > * Some tests fail with 404s which leads me to believe that mirage isn't working
> > 
> > Let me know if I can help clean this up a bit.
> 
> Yeah, sorry ideally we should have merged the Models and Mirage stuff into `develop-future` first. It was tough because I started working on Recommendation stuff before we had the foundational Model and Mirage stuff ready to merge into `develop-future`

Great — could we make this branch the mirage branch? And keep the recommendation component stuff in a separate branch?

Reply:
Hmm I think I'll create a Mirage branch off this branch, and delete the Recc component stuff from that branch. We can merge that branch first